tests/general/IndentationBearTest.py

```diff
@@ -0,0 +1,155 @@
+import unittest
+from queue import Queue
+
+from bears.general.IndentationBear import IndentationBear
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+class IndentationBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section("")
+        self.section.append(Setting('language', 'c'))
+        self.section.append(Setting('language_family', 'c'))
+        self.section.append(Setting('use_spaces', False))
+        self.dep_uut = AnnotationBear(self.section, Queue())
+
+    def get_results(self, file, section=None):
+        if section is None:
+            section = self.section
+        dep_results_valid = self.dep_uut.execute("file", file)
+        uut = IndentationBear(section, Queue())
+        arg_dict = {'dependency_results':
+                    {AnnotationBear.__name__:
+                     list(dep_results_valid)},
+                    'file': file}
+        return list(uut.run_bear_from_section(["file"], arg_dict))
+
+    def verify_bear(self,
+                    valid_file=None,
+                    invalid_file=None,
+                    section=None):
+        if valid_file:
+            valid_results = self.get_results(valid_file, section)
+            self.assertEqual(valid_results, [])
+
+        if invalid_file:
+            invalid_results = self.get_results(invalid_file, section)
+            self.assertNotEqual(invalid_results, [])
+
+    def test_basic_indent(self):
+        valid_file =\
+            ("{\n",
+             "\tright indent\n",
+             "}\n")
+        invalid_file =\
+            ("{\n",
+             "wrong indent\n",
+             "}\n")
+        self.verify_bear(valid_file, invalid_file)
+
+        valid_file2 =\
+            ("a {\n",
+             "\tindent1\n",
+             "\tindent2\n",
+             "}\n")
+
+        invalid_file2 =\
+            ("a {\n",
+             "\tindentlevel1;\n",
+             "\t\tsecondlinehere;\n",
+             "}\n")
+        self.verify_bear(valid_file2, invalid_file2)
+
+    def test_within_strings(self):
+        valid_file1 =\
+            ('"indent withinstring{"\n',
```

Comment:
I'm kind of unhappy with your use of the word "indent", but also maybe I'm wrong.

I understand that an indent is the whitespace in front of a line of code, while `{` is merely a symbol that indicates that an indent is needed for the following line. If this is so I'd like to ask you to search for your use of `"indent"` in this file and rephrase where appropriate (including the commit and PR messages).